app/views/errors/internal_server_error.html.erb

```diff
@@ -1,9 +1,9 @@
 <%= title('Internal Server Error (500)') %>
 
-<p class="text-base text-gold-600">500</p>
+<p class="text-base text-slate-600">500</p>
```

Comment:
Nit: Just noticing we're using slate in the error pages. It's pretty close to the normal gray we use so not a huge deal. But for consistency sake it would be worth changing all instances of slate to gray.